config/default.json

```diff
@@ -10,6 +10,9 @@
   },
   "timemanager": {
     "host": "timemanager-staging.blue-planet.be",
-    "api_key": "placeholder_key"
+    "headers": {
+      "AUTHORIZATION": "placeholder_key"
```

Comment:
Where will you put the actual key eventually when deploying this to staging/production?

Reply:
I'm not sure, what I know is that I didn't want it on github. Moreover, the internet want to suggest ENV variables, but you already told me you disliked that approach, so I'm researching the other options